clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -96,8 +97,9 @@ public void grantAccess(Tester<FileOutStream> tester) {
    */
   public FileOutStream(long fileId, OutStreamOptions options) throws IOException {
     mFileId = fileId;
+    mFileLength = 0;
```

Comment:
Is fileLength necessary? It seems like we will compute it on the master side anyway?